### PR TITLE
Upmerge 14.12.2020 Release Notes Update

### PR DESCRIPTION
**Pull request by de-nordic**
Release notes update pull request.

### CODE REVIEW

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -98,6 +101,8 @@ Bluetooth Mesh
 
   * Time client model callbacks for all message types.
   * Support for the nRF52833 DK in the :ref:`bluetooth_mesh_light` and :ref:`bluetooth_mesh_light_switch` samples.
+  * Added support for Light xyL models.
+  * Disable light control server when manual control has taken effect. This follow Mesh Model Specification section 6.2.3.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:105]:
```suggestion
  * Support for :ref:`bt_mesh_light_xyl_readme`.
  
* Updated the :ref:`bt_mesh_lightness_srv_readme` to disable the light control server when manual control has taken effect. 
  This follows the Mesh Model Specification section 6.2.3.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -222,24 +227,29 @@ The following list summarizes the most important changes inherited from upstream
 * Added support for invoking shell commands (shell management) from the mcumgr command line.
 
 
+Multiprotocol Service Layer (MPSL)
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:230]: MPSL is part of sdk-nrfxlib, so this section should be moved up (line 172/177) and made a subsection (~~~)

File: doc/nrf/releases/release-notes-latest.rst [hunk 3]
```diff
@@ -222,24 +227,29 @@ The following list summarizes the most important changes inherited from upstream
...
+==================================
+
+* The nRF53 has been modified to use SWI by default, instead of EGU.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:233]:
```suggestion
* Modified the nRF5340 to use SWI by default, instead of EGU.
```
What exactly was updated here? MPSL support for nRF5340?
I doubt we updated the chip. ;-)

File: doc/nrf/releases/release-notes-latest.rst [hunk 4]
```diff
@@ -407,6 +423,7 @@ The following list summarizes the most important changes inherited from upstream
     * Deprecated legacy TCP stack (TCP1).
     * Added multiple minor TCP2 bugfixes and improvements.
     * Added network management events for DHCPv4.
+    * Fixed getaddrinfo to actually respect socket type hints.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:426]: Is this covered by https://github.com/nrfconnect/sdk-nrf/pull/3647/files line 484?

File: doc/nrf/releases/release-notes-latest.rst [hunk 5]
```diff
@@ -363,6 +373,12 @@ The following list summarizes the most important changes inherited from upstream
 
     * Added multiple ``nrfx_power``-related fixes to reduce power consumption.
 
+  * PWM:
+
+    * Changed GPIO configuration to Nordic HAL which allows support for GPIO  pins above 31.
+    * Added check if PWM period does not exceed 16-bit value to prevent broken behaviour.
+    * Changed PWM DT configuration to use timer phandle instead of previously used timer-instance.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:380]:
```suggestion
    * Changed the GPIO configuration to use Nordic HAL, which allows support for GPIO pins above 31.
    * Added a check to ensure that the PWM period does not exceed a 16-bit value to prevent erroneous behavior.
    * Changed the PWM DT configuration to use a timer phandle instead of the previously used timer instance.
```

File: doc/nrf/releases/release-notes-latest.rst [hunk 6]
```diff
@@ -435,7 +452,11 @@ The following list summarizes the most important changes inherited from upstream
       * :option:`CONFIG_OPENTHREAD_MAC_SOFTWARE_ACK_TIMEOUT_ENABLE`
       * :option:`CONFIG_OPENTHREAD_MAC_SOFTWARE_RETRANSMIT_ENABLE`
       * :option:`CONFIG_OPENTHREAD_PLATFORM_USEC_TIMER_ENABLE`
-      * :option:`CONFIG_OPENTHREAD_CONFIG_PLATFORM_INFO`
+      * :option:`CONFIG_OPENTHREAD_PLATFORM_INFO`
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:455]: Almost duplicating https://github.com/nrfconnect/sdk-nrf/pull/3647/files

**Review by ru-fu — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-latest.rst (hunk 6 above)
Comment (de-nordic) [reply doc/nrf/releases/release-notes-latest.rst:457]: Unfortunately I do not understand.

File: doc/nrf/releases/release-notes-latest.rst [hunk 7]
```diff
@@ -73,6 +73,7 @@ Thread
 
 * Optimized ROM and RAM used by Thread samples.
 * Disabled Hardware Flow Control on the serial port in :ref:`coap_client_sample` and :ref:`coap_server_sample` samples.
+* Development support for nRF21540 FEM (front-end module) in the Thread samples.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:76]: Please move up to line 73 and indent - this is not an update, but something that has been added.

File: doc/nrf/releases/release-notes-latest.rst [hunk 8]
```diff
@@ -144,6 +149,11 @@ sdk-nrfxlib
 
 See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for the most current information.
 
+Multiprotocol Service Layer (MPSL)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Modified the nRF5340 to use SWI by default, instead of EGU.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:155]: What have we modified here? Surely not the nRF5340 SoC, which is what we're saying at the moment. ;)

File: doc/nrf/releases/release-notes-latest.rst (hunk 6 above)
Comment (ru-fu) [reply doc/nrf/releases/release-notes-latest.rst:454]: Some of these options are also added in PR https://github.com/nrfconnect/sdk-nrf/pull/3647
I don't mind where they are added, but adding them only once would probably prevent merge conflicts. ;)

Comment (de-nordic) [reply doc/nrf/releases/release-notes-latest.rst:454]: > 
> 
> Some of these options are also added in PR #3647
> I don't mind where they are added, but adding them only once would probably prevent merge conflicts. ;)

Ah... right. My bad.

File: doc/nrf/releases/release-notes-latest.rst [hunk 9]
```diff
@@ -144,6 +149,11 @@ sdk-nrfxlib
 
 See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for the most current information.
 
+Multiprotocol Service Layer (MPSL)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Modified the Bluetooth controller for nRF5340 to use SWI by default, instead of EGU.
```
Comment (ru-fu) [inline doc/nrf/releases/release-notes-latest.rst:155]: This is about https://github.com/zephyrproject-rtos/zephyr/pull/27801, correct?
Why is this related to MPSL? To me it looks like an update to the Zephyr Bluetooth Controller.
So it should be moved down to the Zephyr > Drivers > Bluetooth Controller section.
Text suggestion:
Updated the Bluetooth Controller to use SWI instead of EGU for nRF5340.

Comment (de-nordic) [reply doc/nrf/releases/release-notes-latest.rst]: Leaving this to Bluetooth people.

**Review by ru-fu — APPROVED** [review]

**Comment by jfischer-no** [comment]
@de-nordic please rebase

**Review by carlescufi — APPROVED** [review]

**Comment by carlescufi** [comment]
@de-nordic please rebase

**Comment by de-nordic** [comment]
> 
> 
> @de-nordic please rebase

Done, Also during rebase I have found that I am in collision with release notes for OpenThread, where I have been adding information on `CONFIG_OPENTHREAD_PLATFORM_INFO`, which has already been on list in form of `CONFIG_OPENTHREAD_CONFIG_PLATFORM_INFO`, which is not correct option identifier; therefore I have replaced the incorrect one.

**Review by jfischer-no — APPROVED** [review]
